code/modules/spells/aoe_turf/fall.dm

```diff
@@ -72,7 +72,7 @@
 		invocation(user, targets)
 		take_charge(user, skipcharge)
 
-		targets = before_cast(targets)
+		targets = before_cast(targets, user)
```

Comment:
Why does this spell copy perform? Is it literally just for the invocation snowflake which should have been done in invocation()